packages/colors/src/build.js

```diff
@@ -47,13 +51,39 @@ async function build() {
     const variable = `$${PREFIX}-colors__${name}--${grade}`;
     colorsByValue[value] = variable;
 
+    // Collect color names and formatted scss color variables with grades.
+    colorsArray.push({
+      name,
+      grade: `${grade}: ${variable}`,
```

Comment:
Not sure if this is clear enough, but this line `${grade}: ${variable}` comes out to be something like `10: $ibm-colors__red--10` &  it's mean to be the final format of each color grade entry in the color map, like this:
![screen shot 2018-12-11 at 11 53 53 am](https://user-images.githubusercontent.com/9057921/49819583-730e1280-fd3b-11e8-829b-51da1816b609.png)